_data/pratapnea/pratapnea.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Pratap Neaupane # No longer than 18 characters
+institution: Nepathya College # no longer than 58 characters
+profile_pic: pratapnea.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: "Everyone has their own role in someone's life and its really great to meet them. Thank you everyone!" # no longer than 100 characters
```

Comment:
Quote must **not** be longer than 100 characters, yours is **102** characters with the double-quotes. You can choose to remove the double-quotes or make your quote smaller.